en/docs/fragments/local-outbound.md

```diff
@@ -0,0 +1 @@
+5. Expand the Local & Outbound Authentication Configuration section and select Use tenant domain in local subject identifier to sign the ID token with the user's tenant domain.
```

Comment:
**Local & Outbound Authentication Configuration** and **Use tenant domain in local subject identifier** should be in bold